scripts.js

```diff
@@ -0,0 +1,17 @@
+$(function(){
+	var carouselList = $("#carousel ul");
+	setInterval(changeSlide, 3000); 
+
+	function changeSlide(){
+	carouselList.animate({'marginLeft':-400}, 500, moveFirstSlide);
+	}
+	changeSlide();
```

Comment:
you can remove it, in line 3 you start this function in interval every 3 seconds